ai-backend/config/agent_config.py

```
@@ -47,6 +48,33 @@
+    """Configuration for a single agent."""
+    name: str
+    description: str
+    model: str
+    temperature: float
+    max_tokens: int
+    system_prompt: str
+
+
+class AgentConfigurations:
+    """Manages configurations for all agents."""
+    
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)